vis_test.go

```diff
@@ -33,3 +37,13 @@ func TestVis(t *testing.T) {
 		}
 	}
 }
+
+// The resulting string of Vis output could potentially be four times longer than
+// the original. Vis must handle this possibility.
+func TestVisLength(t *testing.T) {
+	testString := "All work and no play makes Jack a dull boy\n"
+	for i := 0; i < 20; i++ {
```

Comment:
Local tests passed on `i<25`, but travis timed out on that and failed that build, so I just changed it to `i<20`. Purpose of the test is still evident, though.